.buildkite/scripts/dra.sh

```diff
@@ -72,6 +72,7 @@ fi
 dra() {
   local workflow=$1
   local command=$2
+  local qualifier=${3:-""}
```

Comment:
This should help with using a default value 

```suggestion
  local qualifier=${3:-"${ELASTIC_QUALIFIER}"}
```